core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -252,7 +261,14 @@ private void tryStart(Profiler profiler) {
 
             logger().info("Creating container for image: {}", dockerImageName);
             profiler.start("Create container");
-            CreateContainerCmd createCommand = dockerClient.createContainerCmd(dockerImageName);
+
+            HostConfig hostConfig = new HostConfig();
```

Comment:
I imagine we will continue adding support for more host config parameters, so I would prefer the `HostConfig` initialization in a private method, like `buildHostConfig()`.